src/test/kotlin/ua/marchenko/artauction/artwork/controller/ArtworkControllerTest.kt

```diff
@@ -0,0 +1,60 @@
+package ua.marchenko.artauction.artwork.controller
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.assertThrows
+import org.mockito.Mockito.*
+import ua.marchenko.artauction.artwork.exception.ArtworkNotFoundException
+import ua.marchenko.artauction.artwork.mapper.toArtwork
+import ua.marchenko.artauction.artwork.mapper.toArtworkResponse
+import ua.marchenko.artauction.artwork.service.ArtworkService
+import ua.marchenko.artauction.common.artwork.getRandomArtwork
+import ua.marchenko.artauction.common.artwork.getRandomArtworkRequest
+import kotlin.test.Test
+
+class ArtworkControllerTest {
+
+    private val mockArtworkService: ArtworkService = mock(ArtworkService::class.java)
+    private val artworkController: ArtworkController = ArtworkController(mockArtworkService)
+
+    @Test
+    fun `getAllArtworks should return a list of ArtworkResponse`() {
+        val artworks = listOf(getRandomArtwork())
+        `when`(mockArtworkService.findAll()).thenReturn(artworks)
+        val result = artworkController.getAllArtworks()
+        assertEquals(1, result.size)
+        assertEquals(artworks[0].toArtworkResponse(), result[0])
```

Comment:
I strongly recommend you to split you code in tests into given-when-then blocks, they will look way more readable with such approach
```suggestion
       // GIVEN
        val artworks = listOf(getRandomArtwork())
        `when`(mockArtworkService.findAll()).thenReturn(artworks)
        
        // WHEN
        val result = artworkController.getAllArtworks()
        
        // THEN
        assertEquals(1, result.size)
        assertEquals(artworks[0].toArtworkResponse(), result[0])
```

Reply:
done